src/osd/osd_types.cc

```diff
@@ -370,6 +373,7 @@ void osd_stat_t::generate_test_instances(std::list<osd_stat_t*>& o)
   o.back()->kb = 1;
   o.back()->kb_used = 2;
   o.back()->kb_avail = 3;
+  o.back()->pg_sum = 100;
```

Comment:
@songbaisen i guess what @liewegas means is the `PGMap::get_num_pg_by_osd(osd)`?

so you don't need to add `pg_sum` to `osd_stat_t`, by the way, the new `pg_sum` field in `osd_stat_t` breaks the compatibility. see `ENCODE_START(4, 2, bl);` in the `osd_stat_t::encode()`.